src/xdotoolify.js

```diff
@@ -680,7 +700,9 @@ _Xdotoolify.prototype.do = async function(options = {unsafe: false}) {
           if (!this.page.xjsLastPos ||
```

Comment:
In hindsight, I think this auto-jitter is probably a mistake due to having hidden unexpected behavior (mousemove sends two events instead of one).

Instead, it should probably throw an error warning that you are trying to mousemove to the same location, and passing a jitter argument if you want to do that. That way when someone is reading the test code, it is very obvious that this mousemove is different than other mousemoves.

Reply:
I removed the jitter from here and put it in its own function.